iOS/Source/SunView.swift

```diff
@@ -151,27 +151,27 @@ class SunView: UIView {
         return SunViewLocation(x: absoluteX, y: absoluteY)
     }
 
-    func updateInterface(controller: DaylightModelController) {
-        self.superview?.backgroundColor = controller.primaryColor
-        self.backgroundColor = controller.primaryColor
-        self.sunriseLabel.textColor = controller.secondaryColor
-        self.sunsetLabel.textColor = controller.secondaryColor
-        self.dayTimeLabel.textColor = controller.secondaryColor
-        self.nightTimeLabel.textColor = controller.secondaryColor
-        self.horizon.backgroundColor = controller.secondaryColor
-        self.sun.tintColor = controller.secondaryColor
-        self.moon.tintColor = controller.secondaryColor
-
-        self.sunViewLocation = self.location(for: controller.percentageInDay)
-        self.moon.isHidden = !controller.shouldShowMoon
-        self.nightTimeLabel.isHidden = !controller.shouldShowMoon
-        self.sun.isHidden = controller.shouldShowMoon
-        self.dayTimeLabel.isHidden = !controller.shouldShowTimeLabel
-
-        self.dayTimeLabel.text = controller.currentTimeString
-        self.nightTimeLabel.text = controller.currentTimeString
-        self.sunriseLabel.text = controller.sunriseTimeString
-        self.sunsetLabel.text = controller.sunsetTimeString
+    func updateInterface(withDaylightModelController daylightModelController: DaylightModelController) {
```

Comment:
this looks really similar to what's happening in the playground - is there any way to pull this into something where you don't have to update it twice?

Reply:
My idea is to play with it in the playground and if it works i copy it over ✌️ 

i think the only we you could directly reuse it will make it harder to flexibly play with it (since then you would have to add the file as a `Resource` and build the project everytime you run the playground, if it's even possible that way!)

Reply:
ah, good point.